lib/dialects/mssql/query-generator.js

```diff
@@ -685,6 +685,11 @@ var QueryGenerator = {
     if (options.limit || options.offset) {
       if (!options.order || (options.include && !orders.subQueryOrder.length)) {
         fragment += (options.order && !isSubQuery) ? ', ' : ' ORDER BY ';
+
+        if(model.name){
+          fragment += this.quoteIdentifier(model.name) + '.';
+        }
```

Comment:
I have made the changes. Thank you kindly for guiding me towards a proper fix. I am still learning the ropes, do let me know if further action is recommended.